test/test_images/helloworld/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Use the offical Golang image to create a build artifact.
+# This is based on Debian and sets the GOPATH to /go.
+# https://hub.docker.com/_/golang
+FROM golang:1.12 as builder
+
+# Copy local code to the container image.
+WORKDIR /go/src/knative.dev/client/test/test_images/helloworld
+COPY . .
```

Comment:
in order to use the default buildah task, we can specify the helloworld apps path here to copy over

Reply:
(or I think only the helloworld.go file, as that's all the build context required)